<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>921</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QListWidget的使用</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>574</width>
      <height>305</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QToolBox" name="toolBox">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>152</width>
        <height>185</height>
       </rect>
      </property>
      <attribute name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/icons/icons/410.bmp</normaloff>:/icons/icons/410.bmp</iconset>
      </attribute>
      <attribute name="label">
       <string>列表项操作</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QToolButton" name="tBtnListIni">
         <property name="text">
          <string>tBtnListIni</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QToolButton" name="tBtnListClear">
         <property name="text">
          <string>tBtnListClear</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QToolButton" name="tBtnListInsert">
         <property name="text">
          <string>tBtnListInsert</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QToolButton" name="tBtnListAppend">
         <property name="text">
          <string>tBtnListAppend</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QToolButton" name="tBtnListDelete">
         <property name="text">
          <string>tBtnListDelete</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>122</width>
        <height>183</height>
       </rect>
      </property>
      <attribute name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/icons/icons/408.bmp</normaloff>:/icons/icons/408.bmp</iconset>
      </attribute>
      <attribute name="label">
       <string>列表排序</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkBox_2">
         <property name="text">
          <string>允许排序</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QToolButton" name="tBtnSortAsc">
         <property name="text">
          <string>升序排序</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
         <property name="arrowType">
          <enum>Qt::UpArrow</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QToolButton" name="tBtnSortDes">
         <property name="text">
          <string>降序排序</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
         <property name="arrowType">
          <enum>Qt::DownArrow</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>57</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <attribute name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/icons/icons/412.bmp</normaloff>:/icons/icons/412.bmp</iconset>
      </attribute>
      <attribute name="label">
       <string>信号发射时机</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QToolButton" name="tBtnClearText">
           <property name="text">
            <string>清空文本</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tBtnAddLine">
           <property name="text">
            <string>添加空行</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>当前项变化</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="editCutItemText"/>
        </item>
        <item>
         <widget class="QCheckBox" name="chkBoxListEditable">
          <property name="text">
           <string>项可编辑</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QToolButton" name="tBtnSelItem">
          <property name="text">
           <string>tBtnSelItem</string>
          </property>
          <property name="popupMode">
           <enum>QToolButton::MenuButtonPopup</enum>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tBtnSelAll">
          <property name="text">
           <string>tBtnSelAll</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tBtnSelNone">
          <property name="text">
           <string>tBtnSelNone</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tBtnSelInvs">
          <property name="text">
           <string>tBtnSelInvs</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget">
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actListini"/>
   <addaction name="actListClear"/>
   <addaction name="actListInsert"/>
   <addaction name="actListAppend"/>
   <addaction name="actListDelete"/>
   <addaction name="separator"/>
  </widget>
  <action name="actListini">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/icons/128.bmp</normaloff>:/icons/icons/128.bmp</iconset>
   </property>
   <property name="text">
    <string>初始化列表</string>
   </property>
   <property name="toolTip">
    <string>初始化列表</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actListClear">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/icons/delete1.bmp</normaloff>:/icons/icons/delete1.bmp</iconset>
   </property>
   <property name="text">
    <string>清除列表</string>
   </property>
   <property name="toolTip">
    <string>清除列表</string>
   </property>
  </action>
  <action name="actListInsert">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/icons/424.bmp</normaloff>:/icons/icons/424.bmp</iconset>
   </property>
   <property name="text">
    <string>插入项</string>
   </property>
   <property name="toolTip">
    <string>插入项</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actListAppend">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/icons/316.bmp</normaloff>:/icons/icons/316.bmp</iconset>
   </property>
   <property name="text">
    <string>添加项</string>
   </property>
   <property name="toolTip">
    <string>添加项</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actListDelete">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/icons/324.bmp</normaloff>:/icons/icons/324.bmp</iconset>
   </property>
   <property name="text">
    <string>删除当前项</string>
   </property>
   <property name="toolTip">
    <string>删除当前项</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actSelAll">
   <property name="text">
    <string>全选</string>
   </property>
   <property name="toolTip">
    <string>全选</string>
   </property>
  </action>
  <action name="actSelNone">
   <property name="text">
    <string>全不选</string>
   </property>
   <property name="toolTip">
    <string>全不选</string>
   </property>
  </action>
  <action name="actSelInvs">
   <property name="text">
    <string>反选</string>
   </property>
   <property name="toolTip">
    <string>反选</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/icons/exit.bmp</normaloff>:/icons/icons/exit.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="actSelPopMenu">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/icons/124.bmp</normaloff>:/icons/icons/124.bmp</iconset>
   </property>
   <property name="text">
    <string>项选择</string>
   </property>
   <property name="toolTip">
    <string>项选择</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actSelPopMenu</sender>
   <signal>triggered()</signal>
   <receiver>actSelInvs</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
